lib/active_merchant/billing/gateways/cyber_source.rb

```diff
@@ -691,7 +691,8 @@ def commit(request, action, amount, options)
         end
 
         success = response[:decision] == "ACCEPT"
-        message = @@response_codes[('r' + response[:reasonCode]).to_sym] rescue response[:message]
+        message = response[:message]
```

Comment:
Not using the normalized error messages is a big change (even if it wasn't always working as intended). This seems unrelated to actually extracting the correct `reasonCode`. Why'd you decide to do this too?